build.gradle

```diff
@@ -43,13 +43,13 @@ buildscript {
 
 ext {
     configDir = new File(rootDir, 'config')
-    jnrUnixsocketVersion = '0.38.13'
-    nettyVersion = '4.1.75.Final'
-    nettyTcnativeBoringsslVersion = '2.0.48.Final'
+    jnrUnixsocketVersion = '0.38.17'
+    nettyVersion = '4.1.79.Final'
+    nettyTcnativeBoringsslVersion = '2.0.54.Final'
     snappyVersion = '1.1.8.4'
-    zstdVersion = '1.5.0-4'
+    zstdVersion = '1.5.2-3'
     mongoCryptVersion = '1.6.0-alpha0'
-    projectReactorVersion = 'Californium-SR23'
+    projectReactorVersion = '2020.0.22'
```

Comment:
Is this a big update / breaking change. I recall last time we were hesitant to use the latest version.

Reply:
Ack